requirements.txt

```diff
@@ -1,7 +1,8 @@
 pandas>=0.23.3,<1
-numpy>=1.14.5,<2
+pymongo>=3.10.1
```

Comment:
is pymongo used in this PR?

Reply:
Not in this PR, but will be in the next once DeepCell Datasets is up and running